### ISSUE TITLE
Can this be used on a custom dataset?

### ISSUE THREAD

**Issue by HoopsMcann** [neulab/InterpretEval#14]

**Comment by neubig** [comment]
Yes. This example from the MasakhaNER dataset may be useful to help you get started: https://github.com/masakhane-io/masakhane-ner/tree/main/analysis_scripts

We're currently in the process of doing major refactoring to make functionality like this be easier to use on custom datasets within ExplainaBoard, but it might be a little while longer: http://explainaboard.nlpedia.ai/
